templates/verify_login.txt

```diff
@@ -1,5 +1,9 @@
 {{t "New sign-in to Firefox"}}
-{{t "For added security, please confirm this sign-in to begin syncing with %(device)s." }}
+{{t "For added security, please confirm this sign-in to begin syncing with this device:" }}
+{{ device }}
+{{t "IP address: %(ip)s" }}
+{{ location }}
```

Comment:
It does:

![screen shot 2016-07-25 at 15 37 01](https://cloud.githubusercontent.com/assets/848085/17105110/aeaa8974-527d-11e6-8216-53201ab4d96c.png)